test/validation/api/event/event.c

```diff
@@ -117,12 +129,20 @@ static int event_types_suite_init(void)
 		ctx->pool_types[n++] = ODP_POOL_VECTOR;
 	}
 
+	if ((dma_capa.compl_mode_mask & ODP_DMA_COMPL_EVENT) &&
+	    dma_capa.pool.max_num > 0) {
+		ctx->event_types[n] = ODP_EVENT_DMA_COMPL;
+		ctx->pool_types[n++] = ODP_POOL_DMA_COMPL;
+	}
+
 	ctx->num_types = n;
 
 	for (i = 0; i < ctx->num_types; i++) {
 		odp_pool_param_t prm;
+		odp_dma_pool_param_t dma_prm;
 
 		odp_pool_param_init(&prm);
+		odp_dma_pool_param_init(&dma_prm);
```

Comment:
Will be called even when DMA is not supported. Better to move this to the switch case.